HighwayPatrol.cpp

```diff
@@ -1,16 +1,21 @@
-#include "HighwayPatrol.h"
 #include <iostream>
+#include "HighwayPatrol.h"
+#include "Highway.h"
+#include "Car.h"
+#include "SemiTruck.h"
+#include "Motorcycle.h"
 
-HighwayPatrol::HighwayPatrol() : Vehicle("HighwayPatrol")
-{
+HighwayPatrol::HighwayPatrol() : Vehicle("HighwayPatrol") { }
 
-}
+HighwayPatrol::~HighwayPatrol() = default;
+HighwayPatrol::HighwayPatrol(const HighwayPatrol&) = default;
+HighwayPatrol& HighwayPatrol::operator=(const HighwayPatrol&) = default;
 
 void HighwayPatrol::scanHighway(Highway* h)
 {
     std::cout << name << ": scanning highway for speeders" << std::endl;
 
-    for( size_t i = h->vehicles.size(); --i >= 0; )
+    for( size_t i = h->vehicles.size(); i--; )
```

Comment:
This should actually be
`for( size_t i = h->vehicles.size(); i-- > 0; )`
please correct it for me.
- [ ] revise